README.md

```diff
@@ -5,7 +5,7 @@ nixos-up is a dead-simple install wizard for NixOS. It's the fastest way to get
 From the NixOS installation USB/CD:
 
 ```
-curl --fail --silent https://raw.githubusercontent.com/samuela/nixos-up/main/nixos-up.ml > nixos-up.ml && chmod +x nixos-up.ml && sudo ./nixos-up.ml
+nix-shell https://github.com/samuela/nixos-up/main.tar.gz
```

Comment:
```suggestion
sudo nix-shell github.com/samuela/nixos-up/archive/refs/heads/main.tar.gz
```
I think we can get rid of the `https://`? I'll have to give this a shot and see if it works.